frontend/src/context/mapStateSlice/index.ts

```diff
@@ -68,7 +68,8 @@ export const layerOrdering = (a: LayerType, b: LayerType) => {
       | 'impact'
       | 'point_data'
       | 'polygon'
-      | 'static_raster']: number;
+      | 'static_raster'
+      | 'composite']: number;
```

Comment:
let's keep them in alphabetical order

Reply:
Do you want all the "order" array to be ordered alphabetical?